applications/vr-grabscale/VRBuildGrabScale.js

```diff
@@ -0,0 +1,496 @@
+//
+//  VRBuildGrabScale.js
+//  
+//  Created by Basinsky on 2/2/2020
```

Comment:
It might have been preferable to include all the original commits (attributed to Basinsky) rather than a single, combined commit.

Reply:
I'm not sure if there's any easy way to do that given the nature of submitting to an entirely new repo.

Reply:
You can create patches from one repo and apply them to another repo.